packages/jsii-pacmak/lib/targets/golang/types/struct.ts

```diff
@@ -85,4 +89,11 @@ export class StructProperty {
     code.closeBlock();
     code.line();
   }
+
+  public get returnType(): string {
```

Comment:
Noop: Just noting for posterity that we should include a comment here about how type resolution stuff has to happen outside the constructor.